app/javascript/mastodon/locales/fr.json

```diff
@@ -368,6 +370,7 @@
   "status.embed": "Intégrer",
   "status.favourite": "Ajouter aux favoris",
   "status.filtered": "Filtré",
+  "status.hide": "Cacher le status",
```

Comment:
Changing translation files manually is generally a bad idea (*expect source files in English*). Translations should be added through Crowdin to avoid problems with synchronization and conflicts during translations update.

I once did that mistake where I deleted some strings from all locales and that introduced conflict in next translation update PR by Crowdin.

Reply:
Ok @Sasha-Sorokin thanks for the feedback, in that case I will remove all french translations

Reply:
I concur, this is likely to cause merge issues.